courses/machine_learning/deepdive2/image_classification/labs/3_tf_hub_transfer_learning.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>We'll fix the link once all of the images notebooks are in master.</p><p> </p><p>Also, 2_mnist_model.ipynb -&gt; 2_mnist_models.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>